src/ngraph/runtime/cpu/mkldnn_utils.hpp

```diff
@@ -112,6 +114,65 @@ namespace ngraph
 
                     return true;
                 }
+
+                template <typename OP>
+                bool can_use_mkldnn_batchnorm(ngraph::Node* node)
```

Comment:
can you split this into two methods instead (can_use_mkldnn_batchnorm_fprop and can_use_mkldnn_batchnorm_bprop)? Doesn't look like it needs to be templated as well.